Deeploy/Targets/Spatz/Platform.py

```
@@ -0,0 +1,119 @@
+]
+
+
+class SpatzEngine(DeploymentEngine):
+    def __init__(self, name: str, Mapping = SpatzMapping, initCode = "", includeList = includeList) -> None:
+        super().__init__(name, Mapping, initCode, includeList)
+
+
+class SpatzPlatform(DeploymentPlatform):
+
+    def __init__( self,
+        engines = [SpatzEngine("SpatzVectorProcessor")],
```

Ruff (0.15.20)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)